core/src/main/java/com/opensymphony/xwork2/security/DefaultExcludedPatternsChecker.java

```diff
@@ -80,7 +89,14 @@ public void setExcludedPatterns(String[] patterns) {
     }
 
     public void setExcludedPatterns(Set<String> patterns) {
-        LOG.trace("Sets excluded patterns [{}]", patterns);
+        if (excludedPatterns != null && excludedPatterns.size() > 0) {
+            LOG.warn("Replacing excluded patterns [{}] with [{}], be aware that this affects all instances and safety of your application!",
+                        excludedPatterns, patterns);
+        }
+        else {
+            LOG.debug("Replacing excluded patterns [{}] with [{}], be aware that this affects all instances and safety of your application!",
```

Comment:
It should be `LOG.debug("Sets excluded patterns [{}]", patterns);`

Reply:
Hello @yasserzamani .
The two `debug` log statements have been updated in the most recent commit based on your comments/request.  I left in the extra warning text information but the wording and parameters of the start of the log output match what you requested.  :)

Reply:
Yes it's great thank you :)

@aleksandr-m , if you consent please approve your review, then we're OK to squash and merge this one :)

Reply:
@yasserzamani Done. Squashing would be great. ;)